pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -132,6 +132,9 @@ func SetDefaults_KubeletConfiguration(obj *kubeletconfigv1beta1.KubeletConfigura
 	if obj.NodeLeaseDurationSeconds == 0 {
 		obj.NodeLeaseDurationSeconds = 40
 	}
+	if obj.MaxContainerCheckpoints == 0 {
+		obj.MaxContainerCheckpoints = 10
```

Comment:
Do I understand correctly that current default is `unlimited` and we're changing it to 10?

Reply:
Exactly. The main goal of this PR is to limit the number of checkpoint archives per container to avoid filling the local disk by creating lot's of checkpoints.

Reply:
First of all it's a user-visible change, so we need a release note. Second, as we're changing current behaviour we need to think twice about possible impact of the change.

Reply:
IMO, as this is ALPHA feature(ContainerCheckpoint is alpha since 1.25), this change is acceptable.

Reply:
@pacoxu Thanks for the explanations! It makes sense to me.

Reply:
Yes, thanks for the clarification. I was also thinking about this but was unsure.

Reply:
Added a comment about unlimited not being supported.